rmw_opendds_cpp/src/rmw_publisher.cpp

```diff
@@ -195,20 +195,20 @@ rmw_create_publisher(
 
     topic = participant->create_topic(
       topic_str, type_name.c_str(),
-      default_topic_qos, NULL, DDS_STATUS_MASK_NONE);
+      default_topic_qos, NULL, DDS::STATUS_MASK_NONE);
```

Comment:
See https://github.com/objectcomputing/OpenDDS/issues/1306, this mask is not yet there in OpenDDS

Reply:
Good find. At this point, I'm only trying to make some changes that are mostly cosmetic. @neeleym is helping me review the real OpenDDS interface. I see that you're correct. We'll add this to our list to fix.